autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,135 @@
+
+debug=yes
+nonInteractive=yes
+
+# Variations in the root disk device name between VMware and Virtualbox
+if [ -e /dev/ada0 ]; then
+    DISKSLICE=ada0
+elif [ -e /dev/da0 ]; then
+    DISKSLICE=da0
+elif [ -e /dev/vtbd0 ]; then
+    DISKSLICE=vtbd0
+else
+    echo "Unknown disk for install.sh to work with!"
+    exit -1
+fi
+
+PARTITIONS="${DISKSLICE} gpt"
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz lib32.txz src.txz ports.txz"
+# DISTRIBUTIONS="kernel.txz base.txz"
+
+
+#!/bin/sh
+set -o xtrace
+
+echo 'Running installerconfig ...' > /dev/ttyu0
+# Enable serial and internal consoles
+echo "Enabling serial and internal consoles" 
+echo "-Dhv" > /boot.config
+echo "cuau0   \"/usr/libexec/getty std.9600\"  xterm   on  secure" >> /etc/ttys
+
+#hostname
+sysrc hostname="FreeBSD"
+
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 da0
+# Set Time Zone to UTC
+echo "Setting Time Zone to UTC ..."
+/bin/cp /usr/share/zoneinfo/UTC /etc/localtime
+/usr/bin/touch /etc/wall_cmos_clock
+/sbin/adjkerntz -a
+
+echo "Set DHCP to NIC ${ifdev} ..." > /dev/ttyu0
+ifdev=$(ifconfig | grep '^[a-z]' | cut -d: -f1 | head -n 1)
+echo "Get ifname ${ifdev}"
+sysrc ifconfig_${ifdev}=DHCP
+sleep 5
+
+# Set latest repository to the new system
+echo "Set latest repository to the new system ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -f | cat
+bit=`uname -m`
+OSFamily=`uname -s`
+major_version=`freebsd-version -k | cut -d . -f 1`
+repo="${OSFamily}:${major_version}:${bit}"
+
+mkdir -p /usr/local/etc/pkg/repos
+cat << EOF > /usr/local/etc/pkg/repos/latest.conf
+FreeBSD: { enabled: no }
+
+latest: {                                                                     
+  url: "pkg+http://pkg.FreeBSD.org/${repo}/latest",
+  mirror_type: "srv",                                                           
+  signature_type: "fingerprints",                                               
+  fingerprints: "/usr/share/keys/pkg",                                         
+  enabled: yes                                                                 
+}
+EOF
+
+# Get DHCP for nic0
+echo "Get IP with dhclient ..." > /dev/ttyu0
+dhclient ${ifdev}
+sleep 15
+echo "Check network ..." > /dev/ttyu0
+ifconfig > /dev/ttyu0
```

Comment:
Why not put this section after setting DHCP for nic0?

Reply:
Change it. Thanks for your suggestion.